lib/mappings/genericEmbeddedMapping.ts

```diff
@@ -0,0 +1,11 @@
+
+// eslint-disable-next-line no-unused-vars
+function embedded (mapping) {
```

Comment:
IMHO it would be better to have a function returning explicitely the corresponding mapping than mutating the argument